metroRequire.d.ts

```diff
@@ -44,6 +44,9 @@ declare namespace __MetroModuleApi {
  * Declare process variable
  */
 declare namespace NodeJS {
-  interface Require extends __MetroModuleApi.RequireFunction {}
+  // Add a custom property to avoid the empty interface issue @typescript-eslint/no-empty-object-type
+  interface Require extends __MetroModuleApi.RequireFunction {
+    customProperty?: string;
+  }
```

Comment:
let's disable this rule for empty interfaces instead, there's an option for that:

- https://github.com/upleveled/eslint-config-upleveled/pull/397

Reply:
Updated ESLint and removed the workaround
[f2c33e6](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/f2c33e6e7317cb16183dd16a79f1845338d6cfbe)